py3status/modules/hddtemp.py

```diff
@@ -0,0 +1,146 @@
+# -*- coding: utf-8 -*-
+"""
+Display hard drive temperatures.
+
+hddtemp is a small utility with daemon that gives the hard drive temperature
+via S.M.A.R.T. (Self-Monitoring, Analysis, and Reporting Technology). This
+module requires the user-defined hddtemp daemon to be running at all times.
+
+Configuration parameters:
+    cache_timeout: refresh interval for this module (default 10)
+    format: display format for this module (default '{format_hdd}')
+    format_hdd: display format for hard drives
+        (default '{name} [\?color=temperature {temperature}°{unit}]')
+    format_separator: show separator if more than one (default ' ')
+    thresholds: specify color thresholds to use
+        *(default [(0, 'deepskyblue'), (25, 'good'),
+        (41, 'degraded'), (46, 'orange'), (51, 'bad')])*
+
+Format placeholders:
+    {format_hdd} format for hard drives
+
+format_hdd placeholders:
+    {name}        name, eg ADATA SP550
+    {path}        path, eg /dev/sda
+    {temperature} temperature, eg 32
+    {unit}        temperature unit, eg C
+
+Temperatures:
+    Less than 25°C: Too cold     (color deepskyblue)
+    25°C to 40°C: Ideal          (color good)
+    41°C to 50°C: Acceptable     (color degraded)
+    46°C to 50°C: Almost too hot (color orange)
+    More than 50°C: Too hot      (color bad)
+
+Color thresholds:
+    temperature: print a color based on the value of hard drive temperature
+
+Requires:
```

Comment:
you also require telnetlib mate

Reply:
Do you have to install that on Gentoo? If yes, I'll add it.